README.md

```diff
@@ -7,32 +7,32 @@
 Terraform Provider for Delphix enables Terraform to create and manage Delphix Continuous Data &
 Continuous Compliance infrastructure using the Delphix Control Tower (DCT) APIs.
 
-Full documentation can he found [here](https://integrations.delphix.com/Terraform/)
+Full implementation directions can be found on the [Delphix Ecosystem Documentation](https://ecosystem.delphix.com/docs/overview-3) and [Terraform Provider Registry](https://registry.terraform.io/providers/delphix-integrations/delphix/latest/docs).
 
 ## Prerequisites
 
-1. Setup Delphix Control Tower(DCT) API Layer. For more information, visit [DCT Home.](https://docs.delphix.com/dct)
-2. Delphix Engines must be registered with DCT APIs
-3. API-Keys must be created for authenticating with DCT APIs. Refer to [DCT Home](https://docs.delphix.com/dct) for more info.
-4. Additional infrastructure required for testing the provider operations [ e.g Hosts to be added as environments, dSources to create VDBs from]
-5. Development setup for GoLang.
+1. Install Delphix Control Tower (DCT). For more information, visit the [DCT documentation](https://docs.delphix.com/dct).
+2. Delphix Continuous Data and Continuous Compliance engines must be connected to DCT.
```

Comment:
QQ: Do we provide any Continuous Compliance with our current terraform provider?

Reply:
Today, it's indirectly. 

For example, a VDB can have a post-hook that triggers a compliance job. But there's nothing today that will run a Job directly.